From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@rcarmo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

carolinacmoravia committed:
C6888: Hardcoded EN locale in link 

Hello, @fhryo-msft,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

ggailey777 committed:
Draft of push zip deployment

ggailey777 committed:
Refactor the shared REST info

ggailey777 committed:
Cleanup and Acrolinx

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ggailey777 committed:
Revising to How To format

kevinlam1 committed:
Revert "update for UI change"

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into s127

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Update Atlassian Cloud tutorial

v-nagta committed:
USER STORY 1140693 SaaS App Tutorial: Vodeclic

v-viinde committed:
USER STORY 1140726 SaaS App Tutorial: FactSet

v-nagta committed:
USER STORY 1140771 SaaS App Tutorial: Cezanne HR Software Tutorial Update

snehaamicrosoft committed:
link for deployment planner

ggailey777 committed:
Added next steps section

ggailey777 committed:
Added customization section

sethmanheim committed:
Rewrite to use .NET Standard open source libraries

sethmanheim committed:
Acrolinx and formatting fixes

sethmanheim committed:
One more Acrolinx fix

ggailey777 committed:
Acrolinx

sethmanheim committed:
Fixes and updating screen shot

sethmanheim committed:
Making headings more consistent

sethmanheim committed:
Typo

sethmanheim committed:
One more typo

sethmanheim committed:
Updates and new screen shots

sethmanheim committed:
Last tweaks

cherylmc committed:
new

sethmanheim committed:
Minor clarification

cherylmc committed:
update

Commit:
rajat11

craigshoemaker committed:
update title, description and h1s for seo

cherylmc committed:
update

sethmanheim committed:
Added PS note

kgremban committed:
Processor note for IoT Core doc

sethmanheim committed:
Fixing note

MikeDodaro committed:
Add info about the insights token and how to get all info except that about recognizedEntities (people) in the image

MikeDodaro committed:
Fix { -> [

ggailey777 committed:
Fixed indent

Commit:
task 629: update caching rules button and screens

chetansriv committed:
Update Atlassin tutorial

sethmanheim committed:
Prettying up the using statements

Commit:
Update ToC link.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into pbie1206

BryanLa committed:
updates

BryanLa committed:
updates

KumudD committed:
moved classic content topics to HowTos

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into UpdateTOCClassic1206

cwatson-cat committed:
Retiring Blitline topic & include file.

Commit:
task 629: fix spacing before graphic

gsacavdm committed:
Add web apps service principal

Commit:
Updated owner to contributor

PRMerger16 committed:
Merge pull request #29451 from gsacavdm/patch-1

Add web apps service principal

BryanLa committed:
updates

PRMerger18 committed:
Merge pull request #29448 from BryanLa/bryanla-msi-adf-links

Bryanla msi adf links

tfitzmac committed:
changed author

Commit:
KRIS_ service fabric metadata update

Commit:
KRIS_SQL-db-metadata update

Commit:
SQL DB metadata update

Lisaco88 committed:
Merge pull request #29405 from sethmanheim/sb-topics

Rewrite to use .NET Standard open source libraries

PRMerger14 committed:
Merge pull request #29455 from kriscrider/mvs-meta2-12.6

KRIS_ service fabric metadata update

PRMerger16 committed:
Merge pull request #29456 from kriscrider/mvc-meta3-12.6

KRIS_SQL-db-metadata update

Lisaco88 committed:
Merge pull request #29444 from Minewiskan/pbie1206

Pbie1206

cherylmc committed:
update

KumudD committed:
fixed terminology and casing

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into UpdateTOCClassic1206

asmalser-msft committed:
Removing reference to obsolete content

PRMerger13 committed:
Merge pull request #29454 from tfitzmac/1206change

changed author

Lisaco88 committed:
Merge pull request #29442 from dksimpson/dean-629

task 629: update caching rules button and screens

rjagiewich committed:
Merge pull request #29322 from chetansriv/atlassiancloud

Atlassiancloud

tfitzmac committed:
delete marketplace app

Lisaco88 committed:
Merge pull request #29452 from cwatson-cat/12-6-retire-blitline

Retiring Blitline topic & include file.

rjagiewich committed:
Merge pull request #29375 from v-nagta/vodeclic

USER STORY 1140693 SaaS App Tutorial: Vodeclic

PRMerger13 committed:
Merge pull request #29438 from MikeDodaro/more-detail-image-srch

More detail image srch

Commit:
rajat11

ggailey777 committed:
another tweak

Commit:
rajat11

rjagiewich committed:
Merge pull request #29462 from tfitzmac/1206delete

delete marketplace app

PRMerger20 committed:
Merge pull request #29453 from raymondlaghaeian/patch-20

Updated owner to contributor

Commit:
automation : correct issue with article being out of date

Lisaco88 committed:
Merge pull request #29457 from kriscrider/mvc-meta4-12.6

KRIS_Mvc meta4 12.6

Lisaco88 committed:
Merge pull request #29449 from KumudD/UpdateTOCClassic1206

Moved class deployment model articles

iainfoulds committed:
Update .openpublishing.redirection.json

Lisaco88 committed:
Merge pull request #29423 from cherylmc/s2ser

new

Lisaco88 committed:
Merge pull request #29426 from MarkusVi/rajat11

rajat11

Lisaco88 committed:
Merge pull request #29469 from iainfoulds/patch-8

VM storage redirects

Lisaco88 committed:
Merge pull request #29378 from v-viinde/factset

USER STORY 1140726 SaaS App Tutorial: FactSet

Lisaco88 committed:
Merge pull request #29383 from v-nagta/cezannev2

USER STORY 1140771 SaaS App Tutorial: Cezanne HR Software Tutorial Up…

asmalser-msft committed:
Update active-directory-saas-google-apps-tutorial.md

Commit:
Merge pull request #29459 from asmalser-msft/patch-1

Removing reference to obsolete content

Commit:
Update iot-hub-csharp-csharp-getstarted.md

PRMerger18 committed:
Merge pull request #29474 from shizn/patch-9

Update iot-hub-csharp-csharp-getstarted.md

mahesh-unnikrishnan committed:
Update active-directory-ds-troubleshooting.md

mahesh-unnikrishnan committed:
Update active-directory-ds-troubleshooting.md

ggailey777 committed:
TOC update for zip push

Commit:
Merge pull request #29475 from mahesh-unnikrishnan/patch-17

Update active-directory-ds-troubleshooting.md

mahesh-unnikrishnan committed:
Update active-directory-ds-faqs.md

mahesh-unnikrishnan committed:
Update active-directory-ds-faqs.md

ggailey777 committed:
tweaks

ggailey777 committed:
fix

pvrk committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Adding another cause from Monthly case analysis

PRMerger10 committed:
Merge pull request #29480 from pvrk/patch-24

Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

PRMerger13 committed:
Merge pull request #29386 from snehaamicrosoft/migrate

added link for deployment planner

spelluru committed:
Azure PowerShell install

veljko-msft committed:
Update sql-database-advisor.md

Deprecating Fix schema issues in favor of Intelligent Insights

veljko-msft committed:
Update sql-database-advisor.md

Commit:
freshness

PRMerger15 committed:
Merge pull request #29486 from spelluru/adfps1207

Azure PowerShell install

Commit:
Merge pull request #29487 from veljko-msft/patch-8

Update sql-database-advisor.md

Commit:
Merge pull request #29489 from Blackmist/freshness

freshness

Commit:
Merge pull request #27191 from carolinacmoravia/patch-438

C6888: Hardcoded EN locale in link

tfitzmac committed:
fixed link

PRMerger18 committed:
Merge pull request #29492 from tfitzmac/1207fixlink

fixed link

craigshoemaker committed:
fix sentence casing in title

tfitzmac committed:
removed deleted tutorial

mithun-prasad committed:
change in name

Commit:
azure auto: integrate with log analytics

PRMerger15 committed:
Merge pull request #29498 from mithun-prasad/patch-7

change in name

craigshoemaker committed:
fix branding typos

Commit:
updates to link

BryanLa committed:
Convert text to note

BryanLa committed:
Update role-based-access-control-resource-provider-operations.md

PRMerger16 committed:
Merge pull request #29499 from georgewallace/autoomsupdate

azure auto: integrate with log analytics

PRMerger17 committed:
Merge pull request #29485 from craigshoemaker/seo-updates

update title, description and h1s for seo

BryanLa committed:
Add link to Azure Resource Manager Resource Provider operations

BryanLa committed:
Update role-based-access-control-resource-provider-operations.md

tfitzmac committed:
fixing link

PRMerger14 committed:
Merge pull request #29505 from tfitzmac/1207applink

fixing link

PRMerger13 committed:
Merge pull request #29502 from BryanLa/patch-16

Add link to Azure Resource Manager Resource Provider operations

Commit:
correcting acrolinx issues so article can be reviewed

cwatson-cat committed:
Deleting unused & outdated include file.

Commit:
Merge pull request #29501 from BryanLa/patch-15

Convert text to note

rjagiewich committed:
Merge pull request #28947 from ggailey777/s127

Add the zip push deployment topic for Functions

v-anpasi committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger16 committed:
Merge pull request #29429 from kgremban/dec6-iotcore

Processor note for IoT Core doc

rjagiewich committed:
Merge pull request #29507 from cwatson-cat/12-7-rmvClassic-appmodel-incl

Deleting unused & outdated include file.

v-anpasi committed:
Merge pull request #29508 from MicrosoftDocs/FromPublicRepo

From public repo

tfitzmac committed:
removed ordered list

PRMerger16 committed:
Merge pull request #29478 from mahesh-unnikrishnan/patch-17

Update active-directory-ds-faqs.md

Lisaco88 committed:
Merge pull request #29466 from georgewallace/autoupdates1206

automation : correct issue with article being out of date

Lisaco88 committed:
Merge pull request #29166 from kevinlam1/revert-28899-create-logic-app

Revert "update for UI change"

Lisaco88 committed:
Merge pull request #29497 from tfitzmac/1207index

removed deleted tutorial
